_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -134,7 +134,7 @@ You will most likely not need to specify any parameters except for `location`. F
    s3.client.default.proxy.port: 8080 # port for your proxy server
    s3.client.default.read_timeout: 50s # the S3 connection timeout
    s3.client.default.use_throttle_retries: true # whether the client should wait a progressively longer amount of time (exponential backoff) between each successive retry
-   s3.client.default.region: us-east-2 # AWS region to use
+   s3.client.default.region: us-east-2 # AWS region to use - for non-AWS S3 storage value is required, but has no effect.
```

Comment:
```suggestion
   s3.client.default.region: us-east-2 # AWS region to use. For non-AWS S3 storage, this value is required but has no effect.
```

Reply:
trying to understand this, if it's non-AWS S3 storage, this required field value needs to be some regions? even though it's not in used?

Reply:
That's correct :)